BUILD.bazel

```diff
@@ -49,6 +50,20 @@ genrule(
     cmd = "mkdir $@",
 )
 
+genrule(
+    name = "copy_extend_oem_bin",
+    srcs = ["//tools/cmd/extend_oem:main_for_build"],
+    outs = ["extend-oem.bin"],
+    cmd = "mv $< $@",
```

Comment:
nit: is there a specific reason why this is "mv" and not "cp"? I'm not sure if genrules like it when the inputs are modified.